_posts/documentation/explore/2000-01-07-related-projects.md

```diff
@@ -67,6 +68,8 @@ Several page capture projects utilizing PhantomJS:
 
  * [Compass Magick](https://github.com/StanAngeloff/compass-magick) is a dynamic image generation for Compass using (among others) PhantomJS.
 
+ * [DeckTape](https://github.com/astefanutti/decktape) is a high-quality PDF exporter from HTML presentation frameworks like impress.js and reveal.js
```

Comment:
Just pick one of the two that is the most suitable/representative. I think it's just fair to everyone to have each tool listed in one category only.